gradle.properties

```diff
@@ -4,3 +4,4 @@ korkVersion=7.16.1
 spinnakerGradleVersion=7.0.1
 enablePublishing=false
 org.gradle.parallel=true
+spring.application.name=orca
```

Comment:
Hmm, why is this necessary? [It should already set by the application itself](https://github.com/spinnaker/orca/blob/master/orca-web/src/main/groovy/com/netflix/spinnaker/orca/Main.groovy#L97).

Reply:
I need to do some digging on this one, not sure why its needed, especially since its in the code already. When running in a container, plugins would fail to be found unless I added this.

Reply:
i also added this to tests here: https://github.com/spinnaker/orca/pull/3405

Reply:
not needed, thank you both!